Makefile

```diff
@@ -50,12 +50,12 @@ DOCS_SYNOPISIS_DIR = docs/source/_tmp
 # Start of the actual build targets
 
 .PHONY: $(GOPATH)/bin/minishift$(IS_EXE)
-$(GOPATH)/bin/minishift$(IS_EXE): vendor $(ADDON_ASSET_FILE)
+$(GOPATH)/bin/minishift$(IS_EXE): $(ADDON_ASSET_FILE) vendor
```

Comment:
+1, This works, there was indeed a bit of an issue when deleting the glide.lock and trying to re-bootstrap. Now it works again :-)